config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -124,6 +124,59 @@ presubmits:
         securityContext:
           privileged: true
 
+  - name: pull-kubernetes-e2e-gce-graceful-shutdown
+    always_run: false
+    cluster: k8s-infra-prow-build
+    skip_branches:
+    - release-\d+\.\d+ # per-release image
+    annotations:
+      testgrid-dashboards: presubmits-kubernetes-nonblocking
+      testgrid-tab-name: pull-kubernetes-e2e-gce-graceful-shutdown
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
+    decorate: true
+    decoration_config:
+      timeout: 80m
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: release
+      base_ref: master
+      path_alias: k8s.io/release
+    spec:
+      containers:
+      - command:
+          - runner.sh
+          - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --build=quick
+        - --cluster=
+        - --env=SHUTDOWN_GRACE_PERIOD=30s
+        - --env=SHUTDOWN_GRACE_PERIOD_CRITICAL_PODS=10s
+        - --extract=local
+        - --gcp-master-image=ubuntu
+        - --gcp-node-image=ubuntu
+        - --gcp-zone=us-west1-b
+        - --ginkgo-parallel=30
+        - --provider=gce
+        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-graceful-shutdown
+        - --test_args=--ginkgo.focus=\[NodeFeature:GracefulNodeShutdown\]
```

Comment:
there are no e2e tests AFAIK
```
grep -r GracefulNodeShutdown test/
test/e2e/nodefeature/nodefeature.go:    GracefulNodeShutdown = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("GracefulNodeShutdown"))
test/e2e/nodefeature/nodefeature.go:    GracefulNodeShutdownBasedOnPodPriority = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("GracefulNodeShutdownBasedOnPodPriority"))
test/e2e/storage/non_graceful_node_shutdown.go:var _ = utils.SIGDescribe(feature.NodeOutOfServiceVolumeDetach, framework.WithDisruptive(), "[LinuxOnly] NonGracefulNodeShutdown", func() {
test/e2e/storage/non_graceful_node_shutdown.go: ginkgo.Describe("[NonGracefulNodeShutdown] pod that uses a persistent volume via gce pd driver", func() {
test/e2e_node/node_shutdown_linux_test.go:var _ = SIGDescribe("GracefulNodeShutdown", framework.WithSerial(), nodefeature.GracefulNodeShutdown, nodefeature.GracefulNodeShutdownBasedOnPodPriority, func() {
test/e2e_node/node_shutdown_linux_test.go:                                                      e2eskipper.Skipf("skipping GracefulNodeShutdown tests as we are running on an old version of systemd : %d", systemdVersion)
test/e2e_node/node_shutdown_linux_test.go:                              string(features.GracefulNodeShutdown):                   true,
test/e2e_node/node_shutdown_linux_test.go:                              string(features.GracefulNodeShutdownBasedOnPodPriority): false,
test/e2e_node/node_shutdown_linux_test.go:                              string(features.GracefulNodeShutdown):                   true,
test/e2e_node/node_shutdown_linux_test.go:                              string(features.GracefulNodeShutdownBasedOnPodPriority): false,
test/e2e_node/node_shutdown_linux_test.go:                              string(features.GracefulNodeShutdown):                   true,
test/e2e_node/node_shutdown_linux_test.go:                              string(features.GracefulNodeShutdownBasedOnPodPriority): true,
```

There are only e2e_node that are not tested by this new job

Reply:
talked offline, this is correct https://github.com/kubernetes/kubernetes/blob/4a1b7daea962b731459c3fa69e152c69bcd460f4/test/e2e/storage/graceful_node_shutdown.go#L41